extensions/scarb-doc/src/lib.rs

```diff
@@ -29,15 +34,35 @@ pub struct AdditionalMetadata {
 pub fn generate_packages_information(
     metadata: &Metadata,
     metadata_for_packages: &[PackageMetadata],
-) -> Vec<PackageInformation> {
+    enabled_features: FeaturesOpts,
+) -> Result<Vec<PackageInformation>, anyhow::Error> {
     let mut packages_information = vec![];
     for package_metadata in metadata_for_packages {
         let authors = package_metadata.manifest_metadata.authors.clone();
 
+        let features_manifest: BTreeMap<FeatureName, Vec<FeatureName>> =
+            TomlManifest::read_from_path(&package_metadata.manifest_path)?
+                .features
+                .unwrap_or_default();
+
+        let initial_cfg_set = initial_cfg_set();
+        let cfg_wtih_features = get_cfg_with_features(
+            initial_cfg_set.clone(),
+            &features_manifest,
+            &enabled_features,
+            false,
+        )?;
```

Comment:
isn't this already available from scarb metadata?

Reply:
scarb metadata cannot determine what features should be enabled, no? I see there is no info about specified features for each package even :/ cc @maciektr

Reply:
Removed it since we are using the features metadata from metadata command.